NetCord/Rest/RestMessage.cs

```diff
@@ -75,67 +78,170 @@ public RestMessage(NetCord.JsonModels.JsonMessage jsonModel, RestClient client)
+    /// Whether this message is pinned in a channel.
+    /// </summary>
     public bool IsPinned => _jsonModel.IsPinned.GetValueOrDefault();
 
+    /// <summary>
+    /// If the message was generated by a <see cref="Webhook"/>, this is the webhook's ID.
+    /// </summary>
     public ulong? WebhookId => _jsonModel.WebhookId;
 
+    /// <summary>
+    /// The type of the message.
+    /// </summary>
     public MessageType Type => _jsonModel.Type.GetValueOrDefault();
 
+    /// <summary>
+    /// Sent with Rich Presence-related chat embeds.
+    /// </summary>
     public MessageActivity? Activity { get; }
 
+    /// <summary>
+    /// Sent with Rich Presence-related chat embeds.
+    /// </summary>
     public Application? Application { get; }
 
+    /// <summary>
+    /// If the message is an <see cref="IInteraction"/> or application-owned <see cref="Webhook"/>, this is the ID of the <see cref="NetCord.Application"/>.
+    /// </summary>
     public ulong? ApplicationId => _jsonModel.ApplicationId;
 
+    /// <summary>
+    /// Contains data showing the source of a crosspost, channel follow add, pin, or message reply.
+    /// </summary>
     public MessageReference? MessageReference { get; }
 
+    /// <summary>
+    /// A <see cref="MessageFlags"/> object indicating the message's applied flags.
+    /// </summary>
     public MessageFlags Flags => _jsonModel.Flags.GetValueOrDefault();
 
+    /// <summary>
+    /// The message associated with the <see cref="MessageReference"/>.
+    /// </summary>
+    /// <remarks>
+    /// This field is only returned for messages with a <see cref="MessageType"/> of <see cref="MessageType.Reply"/> or <see cref="MessageType.ThreadStarterMessage"/>. If the message is a reply but the <see cref="ReferencedMessage"/> field is not present, the backend did not attempt to fetch the message that was being replied to, so its state is unknown. If the field exists but is null, the referenced message was deleted.
+    /// </remarks>
     public RestMessage? ReferencedMessage { get; }
 
+    /// <summary>
+    /// Sent if the message is sent as a result of an <see cref="IInteraction"/>.
+    /// </summary>
     public MessageInteractionMetadata? InteractionMetadata { get; }
 
+    /// <summary>
+    /// Sent if the message is a response to an <see cref="IInteraction"/>.
+    /// </summary>
     [Obsolete($"Replaced by '{nameof(InteractionMetadata)}'")]
     public MessageInteraction? Interaction { get; }
 
+    /// <summary>
+    /// The <see cref="GuildThread"/> that was started from this message.
+    /// </summary>
     public GuildThread? StartedThread { get; }
 
+    /// <summary>
+    /// A list of <see cref="IMessageComponent"/> objects, sent if the message contains components like <see cref="Button"/>s, <see cref="ActionRow"/>s, or other interactive components.
+    /// </summary>
     public IReadOnlyList<IMessageComponent> Components { get; }
 
+    /// <summary>
+    /// Sent if the message contains stickers.
```

Comment:
I don't think the doc is sufficient. It's never null so it doesn't make sense I think

Reply:
Corrected then.